android/tangram/jni/platform_android.cpp

```diff
@@ -245,20 +256,25 @@ unsigned char* bytesFromFileSystem(const char* _path, unsigned int* _size) {
 
 }
 
-std::string stringFromFile(const char* _path, PathType _type) {
+std::string stringFromFile(const char* _path, PathType _type, const char* _resourceRoot) {
 
     unsigned int length = 0;
-    unsigned char* bytes = bytesFromFile(_path, _type, &length);
+    unsigned char* bytes = bytesFromFile(_path, _type, &length, _resourceRoot);
     std::string out(reinterpret_cast<char*>(bytes), length);
     free(bytes);
 
     return out;
 
 }
 
-unsigned char* bytesFromFile(const char* _path, PathType _type, unsigned int* _size) {
+unsigned char* bytesFromFile(const char* _path, PathType _type, unsigned int* _size, const char* _resourceRoot) {
```

Comment:
It was in the scene at some point https://github.com/tangrams/tangram-es/pull/776/commits/bb0a0570e0d12b7f16a781158bcd4477570a49c9, how was that an issue?